.github/workflows/Linux_x86_64_SDL1.yml

```diff
@@ -0,0 +1,40 @@
+name: Linux_x86_64_SDL1
+
+on: [push, pull_request]
```

Comment:
Is there a way where we can have it only trigger once for the PR and once for the merged commit in master?

The current rule makes it trigger twice on PR and once for master.

Reply:
I just copied the one from the Mac build. It was like that. And when I looked at the pr from when it was merged in and you had him change it to that so that pushes and pr's would get builds

Reply:
yeah i know, it was not ideal, but better then when it only checked after merging. Running to builds instead of one inst a huge deal, but now it would result in 8 instead of 4 which will probably cause things to queue up so we should fix it now :)